test/IcalCommandTest.py

```diff
@@ -80,5 +80,31 @@ def test_help_python32(self):
         self.assertEqual(self.output, "")
         self.assertEqual(self.errors, "icalendar is not supported in this Python version.\n")
 
+class IcalCommandUnicodeTest(CommandTest):
+    def setUp(self):
+        super(IcalCommandUnicodeTest, self).setUp()
+        self.todolist = load_file_to_todolist("test/data/ListCommandUnicodeTest.txt")
+
+    @unittest.skipIf(IS_PYTHON_32, "icalendar is not supported for Python 3.2")
+    def test_ical_unicode(self):
+        def replace_ical_tags(p_text):
```

Comment:
Please move this function outside the class, and call it from here and from `IcalCommandTest.test_ical()`. This reduces code duplication.